src/bin/wasmer.rs

```diff
@@ -113,6 +114,7 @@ fn main() {
     let options = CLIOptions::from_args();
     match options {
         CLIOptions::Run(options) => run(options),
+        #[cfg(not(target_os = "windows"))]
         CLIOptions::SelfUpdate => update::self_update(),
```

Comment:
Rather than removing this option from the CLI I would prefer to raise an Error on the `self_update` function itself.

The API that we deliver in our CLI needs to be consistent across platforms.